examples/ir-ilu-preconditioned-solver/ir-ilu-preconditioned-solver.cpp

```diff
@@ -76,8 +76,16 @@ int main(int argc, char *argv[])
 
     // Read data
     auto A = gko::share(gko::read<mtx>(std::ifstream("data/A.mtx"), exec));
-    auto b = gko::read<vec>(std::ifstream("data/b.mtx"), exec);
-    auto x = gko::read<vec>(std::ifstream("data/x0.mtx"), exec);
+    // Create RHS and initial guess as 1
+    gko::size_type size = A->get_size()[0];
+    auto host_x = vec::create(exec->get_master(), gko::dim<2>(size, 1));
+    for (auto i = 0; i < size; i++) {
+        host_x->at(i, 0) = 1.;
+    }
+    auto x = vec::create(exec);
+    auto b = vec::create(exec);
+    x->copy_from(host_x.get());
```

Comment:
use x = zeros before but ones now. Is it correct?
if it changes the result, you also need to change the result.dox.

Reply:
Ones is the choice we took for the EuroPar paper, so it is correct now. I need to change the result.dox, thanks!